src/components/styles/Footer.scss

```diff
@@ -6,4 +7,19 @@
 
 .link {
     color: white;
+}
+
+.theme {
```

Comment:
looks great!! can this also be applied to the links on the home page?

Reply:
Yup, good call.